programs/stake/src/stake_instruction.rs

```diff
@@ -261,13 +261,8 @@ pub fn split(
     split_stake_pubkey: &Pubkey,
 ) -> Vec<Instruction> {
     vec![
-        system_instruction::create_account(
-            authorized_pubkey, // Sending 0, so any signer will suffice
-            split_stake_pubkey,
-            0,
-            std::mem::size_of::<StakeState>() as u64,
-            &id(),
-        ),
+        system_instruction::allocate(split_stake_pubkey, std::mem::size_of::<StakeState>() as u64),
+        system_instruction::assign(split_stake_pubkey, &id()),
```

Comment:
This will break the Ledger app's parser.  Splits will need to be blind signed until it's updated

Reply:
Looks like it used to be allocate + assign, then something changed https://github.com/LedgerHQ/app-solana/blob/1c72216edf4e5358f719b164a8d1b6100988b34d/libsol/transaction_printers.c#L29

Reply:
Ah right.  Good ole lazy past-Trent saves the day :wink:

Reply:
That's a rarity.  I don't think past me has ever helped me that much